docs/management/admin/response-actions.asciidoc

```diff
@@ -111,6 +111,29 @@ TIP: You can use the <<use-osquery,Osquery manager integration>> to query a host
 When {elastic-defend} prevents file activity due to <<malware-protection,malware prevention>>, the file is quarantined on the host and a malware prevention alert is created. To retrieve this file with `get-file`, copy the path from the alert's *Quarantined file path* field (`file.Ext.quarantine_path`), which appears under *Highlighted fields* in the alert details flyout. Then paste the value into the `--path` parameter. 
 ====
 
+=== `execute`
+
+Run a shell command on the host. The command's output and any errors are returned in the response console, up to 2000 characters. The complete output (stdout and stderr) are also saved to a downloadable .zip archive (password: `elastic`). Use these parameters:
+
+* `--command` : (Required) A single-line command to run on the host. The command must be supported by the host operating system's default terminal shell (for example, bash for Linux, cmd.exe for Windows).
```

Comment:
I'm not clear on what single-line means here. Is there a character limit? Would it be accurate to just say "A shell command to run on the host"?

Reply:
That might work, but I'm not certain. This is meant to say that the user can't run commands that require multiple lines (or returns, newlines, etc.?). Everything you want to do has to be on a single command line: either a single command, or multiple commands piped or concatenated or otherwise combined on a single line. I was trying to convey that as concisely as possible, without too much exposition.

@paul-tavares @kevinlog WDYT?

Reply:
I'm ok with either the current value or the suggestion by @benironside .

The thing I would probably remove is "default terminal shell" and instead just mention `bash` for linux and Mac, and `cmd.exe` for Windows. Thats because the default terminal currently on Mac is `zsh`

Reply:
@paul-tavares Are you saying that this response action will use bash on a macOS system, even if its default is actually zsh?

I was thinking "default terminal shell" could help future-proof in case one of these systems changes their default, like macOS did a few years ago when they went from bash to zsh.

Reply:
Correct. that is my understanding. We had decided to use specific shells since we really can't control what the user (the user id that the command is executed as) has setup as their default shell.

Reply:
```suggestion
* `--command` : (Required) A shell command to run on the host. The command must be supported by `bash` for Linux and macOS hosts, and `cmd.exe` for Windows).
```

Reply:
LGTM 👍